kubernetes/compose/v1beta1/stack_types.go

```diff
@@ -1,27 +1,45 @@
-package v1beta1
+package v1beta1 // import "github.com/docker/cli/kubernetes/compose/v1beta1"
 
 import (
+	"encoding/json"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
 // StackList defines a list of stacks
-// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 type StackList struct {
 	metav1.TypeMeta `json:",inline"`
 	metav1.ListMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 
 	Items []Stack `json:"items" protobuf:"bytes,2,rep,name=items"`
 }
 
-// +genclient=true
-// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// DeepCopyObject clones the stack list
+func (s *StackList) DeepCopyObject() runtime.Object {
+	if s == nil {
+		return nil
+	}
+	result := new(StackList)
+	result.TypeMeta = s.TypeMeta
+	result.ListMeta = s.ListMeta
+	if s.Items == nil {
+		return result
+	}
+	result.Items = make([]Stack, len(s.Items))
+	for ix, s := range s.Items {
+		result.Items[ix] = *s.clone()
+	}
+	return result
+}
 
 // Stack defines a stack object to be register in the kubernetes API
-// +k8s:openapi-gen=true
-// +resource:path=stacks,strategy=StackStrategy
-// +subresource:request=Owner,path=owner,rest=OwnerStackREST
 type Stack struct {
+	StackImpl
```

Comment:
I don't think this is required at all 🤔 (there shouldn't be any code change in v1beta1)